operational_analysis/methods/turbine_long_term_gross_energy.py

```diff
@@ -157,14 +160,22 @@ def filter_turbine_data(self):
             # Apply range filter
             dic[t].loc[:,'flag_range'] = filters.range_flag(dic[t].loc[:, 'wmet_wdspd_avg'], below = 0, above = 40)
             # Apply frozen/unresponsive sensor filter
-            dic[t].loc[:,'flag_frozen'] = filters.unresponsive_flag(dic[t].loc[:, 'wmet_wdspd_avg'], threshold = 3)
+            dic[t].loc[:,'flag_frozen'] = filters.unresponsive_flag(dic[t].loc[:, 'wmet_wdspd_avg'], threshold = self._rep_threshold)
             # Apply window range filter
             dic[t].loc[:,'flag_window'] = filters.window_range_flag(window_col = dic[t].loc[:, 'wmet_wdspd_avg'], 
-                                                                    window_start = 5., 
+                                                                    window_start = 5, 
                                                                     window_end = 40, 
                                                                     value_col = dic[t].loc[:, 'wtur_W_avg'], 
-                                                                    value_min =  0.02*turb_capac,
+                                                                    value_min =  0.01*turb_capac,
                                                                     value_max =  1.2*turb_capac) 
+
+            # Apply window range filter; note flipped axes approach
+            dic[t].loc[:,'flag_window_2'] = filters.window_range_flag(window_col = dic[t].loc[:, 'wtur_W_avg'], 
+                                                                    window_start = .97*turb_capac, 
+                                                                    window_end = 1.2*turb_capac, 
+                                                                    value_col = dic[t].loc[:, 'wmet_wdspd_avg'],
+                                                                    value_min =  8,
+                                                                    value_max =  30)
```

Comment:
Where do these numbers come from? Are they standard? If so, please cite where you're getting them from. Or, at least, write these constants in the docstring. If they're not standard, consider parametrizing.